recipe/meta.yaml

```diff
@@ -12,25 +12,23 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   skip: true  # [py<38]
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
   host:
-    - cython
-    - numpy
-    - pip
-    - scikit-learn >=1.2.0,<1.3
-    - setuptools_scm
-    - packaging
+    - cython 0.29.32
+    - numpy {{ numpy }}
+    - scikit-learn 1.2.1
+    - setuptools_scm 7.0.4
+    - packaging 
     - python
+    - pip
+    - wheel
+    - setuptools
   run:
     - python
     - {{ pin_compatible('numpy') }}
```

Comment:
`    - importlib_resources >=1.3  # [py<39]` instead of    ` - importlib_resources >=1.3  # [py <= 38]`

Reply:
But this is the same thing.